hunt.py

```diff
@@ -22,163 +22,185 @@
 import lib.calendar as gcalendar
 
 if __name__ == "__main__":
-
     banner()
-    
     # We change the current working directory to allow using GHunt from anywhere
     os.chdir(Path(__file__).parents[0])
-
     if len(sys.argv) <= 1:
         exit("Please put an email address.")
```

Comment:
zaidepaullucas@gmail.com

Reply:
Hgsanders81@gmail.com

Reply:
Why are you commenting email adresses ?

Reply:
No idea

Reply:
all honestest , I have no idea what I'm doing on here. My apologies. I would love to use this email thing you got going on tho.

Reply:
Just clone the repository and follow the instructions for the requirements and usage, good luck